UnitsNet.Tests/CustomCode/StandardVolumeFlowTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by \generate-code.bat.
+//
+//     Changes to this file will be lost when the code is regenerated.
+//     The build server regenerates the code before each build and a pre-build
+//     step will regenerate the code on each local build.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add UnitDefinitions\MyQuantity.json and run generate-code.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class StandardVolumeFlowTests : StandardVolumeFlowTestsBase
+    {
+        // Override properties in base class here
+        protected override bool SupportsSIUnitSystem { get; }
+        protected override double StandardCubicCentimetersPerMinuteInOneStandardCubicMeterPerSecond => 6e7;
+        protected override double StandardCubicFeetPerHourInOneStandardCubicMeterPerSecond => 1.271328001973604e+5;
+        protected override double StandardCubicFeetPerMinuteInOneStandardCubicMeterPerSecond => 2.11888E3;
+        protected override double StandardCubicFeetPerSecondInOneStandardCubicMeterPerSecond => 35.314666721489;
+        protected override double StandardCubicMetersPerDayInOneStandardCubicMeterPerSecond => 8.64e4;
+        protected override double StandardCubicMetersPerHourInOneStandardCubicMeterPerSecond => 3.6e3;
+        protected override double StandardCubicMetersPerMinuteInOneStandardCubicMeterPerSecond => 6.0e1;
+        protected override double StandardCubicMetersPerSecondInOneStandardCubicMeterPerSecond => 1;
+        protected override double StandardLitersPerMinuteInOneStandardCubicMeterPerSecond => 60000;
```

Comment:
Verified all these on Google, by removing "Standard" from the search and treating as volumetric flow units.